src/store/middlewares/network.js

```diff
@@ -18,7 +18,7 @@ const getServerUrl = (nodeUrl, nethash) => {
   if (/\.(liskdev.net|lisk.io)$/.test(nodeUrl)) {
     return nodeUrl.replace(/\.(liskdev.net|lisk.io)$/, $1 => `-service${$1}`);
   }
-  return 'unavailable';
+  return 'http://localhost:9091';
```

Comment:
is this a permanent change or should be removed later?

Reply:
We should figure out a better way. For now, this saves time.